### ISSUE TITLE
find_circular_dependencies.py broken

### ISSUE THREAD

**Issue by gold2718** [ESMCI/cime#1701]
find_circular_dependencies.py was broken by commit 801a8f19b255f65639bda91d2544f63e61cbfddd

**Comment by gold2718** [comment]
@erichlf, can you please fix this?

**Comment by erichlf** [comment]
Yeah, sorry it is taking me this long to get to it. I will get on it.

**Comment by erichlf** [comment]
@gold2718 @jgfouca Since it wasn't caught in my initial testing, how should I test if the code is correct?

**Comment by gold2718** [comment]
It's a python script, run it :)
```
<cimeroot>/tools/utils/find_circular_dependency.py <bldroot>/atm/obj/Depends 
```

**Comment by rljacob** [comment]
Why was this closed?

**Comment by erichlf** [comment]
Accidental

**Comment by gold2718** [comment]
Why was this assigned? It was closed (fixed) in PR #1755.
Is it broken somewhere? I have used it recently with no issues.

**Comment by jgfouca** [comment]
@gold2718 , I assigned to you so you would get credit for the fix.
